src/keys/get-management-token.ts

```diff
@@ -16,6 +16,8 @@ export interface GetManagementTokenOptions {
   reuseToken?: boolean
 }
 
+const defaultCache = new NodeCache()
```

Comment:
What about 

```
let defaultCache: NodeCache;

export const getManagementToken = (privateKey: string, opts: GetManagementTokenOptions) => {
  if (opts.reuseToken) {
    defaultCache = new NodeCache();
  }
  ...
```

So we don't over-allocate if it's not needed. We could also cleanup `createGetManagementToken` at that point